.prettierignore

```diff
@@ -0,0 +1,7 @@
+**/coverage/**
+**.graphql
+.graphqlconfig
+
+# breaks prettier graphql parser
+packages/graphql-language-service-server/src/__tests__/__queries__/test.graphql
+packages/codemirror-graphql/src/__tests__/schema-kitchen-sink.graphql
```

Comment:
Maybe we should ignore all the graphql files in `__tests__`; some of them might test whitespace?

Reply:
good point... that reminds me of at least one test "fix" i had to make that was suspicious in this PR...